eli5/sklearn/permutation_importance.py

```diff
@@ -151,6 +154,12 @@ def __init__(self, estimator, scoring=None, n_iter=5, random_state=None,
         self.cv = cv
         self.rng_ = check_random_state(random_state)
 
+    def wrap_scorer(self, base_scorer, pd_columns):
```

Comment:
would you mjind making this method private (`_wrap_scorer`)? I'm not sure it should be a part of the public API